src/jquery.sidebar.js

```diff
@@ -114,7 +112,15 @@
          * */
         self.on("sidebar:close", function(callback) {
             var properties = {};
-            properties[settings.side] = settings.range[0];
+            if (settings.side == "left" || settings.side == "right")
```

Comment:
@teamrc As @gabipetrovay pointed out, just keep pushing in this branch and the PR will be updated. You can [edit the file on GitHub](https://github.com/teamrc/jQuery-sidebar/edit/sidebar-resize/src/jquery.sidebar.js/) and commit the fixes.